nsiqcppstyle_exe.py

```diff
@@ -40,7 +38,7 @@
 import updateagent.agent
 from nsiqcppstyle_util import *
 
-version = "0.3.0.1"
+version = "0.3.0.2"
```

Comment:
Addition of feature = increase minor version for semver and patch for 0-ver. Let's bump this to 0.3.1

Reply:
Done.